compiler/rustc_parse/src/parser/expr.rs

```diff
@@ -276,7 +276,7 @@ impl<'a> Parser<'a> {
                 lhs = self.parse_assoc_op_ascribe(lhs, lhs_span)?;
                 continue;
             } else if op == AssocOp::DotDot || op == AssocOp::DotDotEq {
-                // If we didn’t have to handle `x..`/`x..=`, it would be pretty easy to
+                // If we did not have to handle `x..`/`x..=`, it would be pretty easy to
```

Comment:
this one seems kinda arbitrary, but ok

Reply:
i just realized this was a backtick, that's why spellcheck got confused. I'll change it back to a regular tick.